src/firefly/java/edu/caltech/ipac/firefly/server/LocalFSWorkspace.java

```diff
@@ -79,6 +80,11 @@ public WsResponse getList(String parentUri, int depth) throws IOException {
     }
 
     @Override
+    public WsResponse getFile(String fileUri, Consumer consumer) throws WsException {
+        throw new IllegalArgumentException("not implemented");
+    }
```

Comment:
Consumer should be `Consumer<InputStream>`.
And if you want to implement it, it's pretty straight forward.  Move logic to new method, then delegate old one like below.
```
    public void getFile(String fileUri, File outputFile) throws WsException {
        getFile(fileUri, inStream -> {
            try {
                FileUtil.writeToFile(inStream, outputFile);
            } catch (IOException e) {
                throw new WsException(e);
            }
        });
    }
```